### DISCUSSION TITLE
github sponsors

### DISCUSSION THREAD

**Discussion by umlx5h** [SuRGeoNix/Flyleaf#479] [General]
Thanks for your hard work.
I would like to support your project, do you have plan to accept github sponsors or others?

**Comment by SuRGeoNix** [comment]
Thanks @umlx5h! Till now it was just fun and hobby/testing so I wouldn't consider sponsorship.
Now that the current design reach it's limits I will try to provide a more professional library (still hobby though).
So, in major 4 I will consider sponsorship and possible a 'free for non commercial use' license

**Reply by umlx5h** [reply]
thanks for reply, looking forward to the new version!
